beluga/include/beluga/sensor/likelihood_field_model.hpp

```diff
@@ -58,6 +58,10 @@ struct LikelihoodFieldModelParam {
    * Used to calculate the probability of the obstacle being hit.
    */
   double sigma_hit = 0.2;
+  /// Pre-computed distance map for unkwnown spaces
+  double unknown_distance_map =
+      -2 * sigma_hit * sigma_hit *
+      std::log(std::sqrt(2 * Sophus::Constants<double>::pi()) * (1 - z_random) / max_laser_distance / z_hit);
```

Comment:
@DPR00 consider using the same auxiliary variables as for the forward calculation for readability.